frontend/src/components/SignUp/SignUp.js

```diff
@@ -60,11 +60,12 @@ class SignUp extends Component {
                   name="firstName"
                   className="sign-in-input firstName col-md form-control-lg"
                   placeholder="First"
+                  required
                   onFocus={event => (event.target.setAttribute('placeholder', ''))}
                   onBlur={event => (event.target.setAttribute('placeholder', 'First'))}
```

Comment:
Should we change these placeholders, as well?

Reply:
I know it's weird, but the design doc has "First" and "Last" as the placeholders and "First Name" and "Last Name" as the popup text. No changes are needed to that.

Reply:
@vhoof If you have questions about the design/product, please mention me as I usually assume questions are for the devs

Reply:
It was intended for him, but that's cool.